plugins/morph_resultsdb.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import logging
+import logging.config
+import json
+import time
+import os
+import requests
+
+from lib.logging_conf import setup_logging
```

Comment:
Yes but also no. In current state no metamoprh folder exists so it's correct. I'll solve it in a minute.